packages/kbn-unsaved-changes-prompt/src/unsaved_changes_prompt/unsaved_changes_prompt.test.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+jest.mock('react-router-dom', () => ({
+  Prompt: () => null,
+}));
+import { Prompt } from 'react-router-dom';
+
+import { shallow, mount, ReactWrapper } from 'enzyme';
+
+import { UnsavedChangesPrompt } from './unsaved_changes_prompt';
+
+describe('UnsavedChangesPrompt', () => {
```

Comment:
Should we add any tests within the ingest pipelines plugin itself, or do you think this is sufficient?

Reply:
Added an extra test with in the plugin iteself with https://github.com/elastic/kibana/pull/183699/commits/f66060b3d834807f58158d0ab07392106eea810d